tutorials/custom_oauth_client_id.md

```diff
@@ -98,24 +98,40 @@ Download the [template](../protocols/profiles/google_client_id.md#sample-google-
 - {download}`Google Storage Custom OAuth Client ID.cyberduckprofile<../protocols/profiles/_static/Google Storage Custom OAuth Client ID.cyberduckprofile>`
 
 
-Use the _OAuth Client ID_ created to edit the `OAuth Client ID` and `OAuth Redirect Url` in the template connection profile leaving other keys unchanged.
+Use the _OAuth Client ID_ created to edit the `OAuth Client ID`, `OAuth Redirect Url`, and optional the `OAuth Client Secret` in the template connection profile leaving other keys unchanged.
```

Comment:
- No comma before `and`.
- `optional`→ `optionally`

Reply:
Based on Grammarly, the comma before the `and` there belongs there because of the enumeration